wwwapp/migrations/0059_migrate_away_from_python_allaccess.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-03-07 06:29
+from __future__ import unicode_literals
+
+from django.db import migrations
+from allaccess.models import AccountAccess
+from social_django.models import UserSocialAuth
+
+# After the migration is applied in production just comment out code depending on the allaccess library
+def forwards_func(apps, schema_editor):
+    for access in AccountAccess.objects.all():
+        if access.provider.name == 'facebook':
+            UserSocialAuth(provider='facebook', uid=access.identifier, user=access.user, extra_data=access.access_token).save()
+        elif access.provider.name == 'google':
+            UserSocialAuth(provider='google-oauth2', uid=access.user.email, user=access.user, extra_data=access.access_token).save()
+
+def reverse_func(apps, schema_editor):
+    raise RuntimeError("Migration not reversible")
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wwwapp', '0058_auto_20200211_1019'),
```

Comment:
This should also reference the allaccess and django-social-auth migrations

https://docs.djangoproject.com/en/3.0/topics/migrations/#accessing-models-from-other-apps